app/views/blocks/radioQuestion.html

```diff
@@ -15,7 +15,7 @@
       show-errors
       name="{{ block.id }}"
       ng-required="block.required"
-      ng-model="answer.value"
+      ng-model="$parent.selectedAnswer"
```

Comment:
Do you actually need to specify `$parent`?

Reply:
Yes, otherwise it tries to read/write the `selectedAnswer` property on the new scope created by `ng-repeat`.